client/client.go

```diff
@@ -49,6 +52,15 @@ func newClient(opts Options) (*client, error) {
 	return &client{opts: opts, newSessionFn: newSession}, nil
 }
 
+func (c *client) Options() (AdminOptions, error) {
```

Comment:
I would name this `AdminOptions()` and also add a corresponding `Options()` that returns just `Options`.